server/controllers/connectedAccounts.js

```diff
@@ -131,6 +132,38 @@ export const createOrUpdate = (req, res, next, accessToken, data, emails) => {
   }
 };
 
+export const disconnect = async (req, res) => {
+  const { collectiveId: CollectiveId, service } = req.params;
+  const { remoteUser } = req;
+
+  try {
+    mustBeLoggedInTo(remoteUser, 'disconnect this connected account');
+
+    if (!remoteUser.isAdmin(CollectiveId)) {
+      throw new errors.Unauthorized({
+        message: 'You are either logged out or not authorized to disconnect this account',
+      });
+    }
+
+    const account = await ConnectedAccount.findOne({
+      where: { service, CollectiveId },
+    });
+
+    await account.delete();
```

Comment:
Let's be safe

```suggestion
    if (account) {
        await account.destroy();
    }
```

Reply:
resolved with https://github.com/opencollective/opencollective-api/pull/2672/commits/6e3ad30b03e212430eb30a015eb716703ec39d96